src/frontend/src/pages/index.css

```diff
@@ -12,6 +12,11 @@ html {
   /*translates to 10px -> root font*/
 }
 
+.MuiPaper-root.MuiCard-root.MuiPaper-elevation1.MuiPaper-rounded {
+  font-size: 100%;
+  height: auto;
+}
+
```

Comment:
The unwanted styling described in #741 is still being applied to the My Feeds component:

![notfixed](https://user-images.githubusercontent.com/13500769/75643738-e5536700-5c0d-11ea-977b-68ee8bfd0e23.gif)

Reply:
As I've mentioned in the description of the pr I removed styling only from myfeeds block but I did not touch the body of the page. I believe that in the gif that you've sent you're testing the body of the page. I cannot remove this styling from the body by editing `index.css` because gatsby did not generate the id for this specific body.

Reply:
If that's the case, you may need to tailor your approach, e.g. making changes to existing CSS selectors (instead of adding another styling rule) to exempt the My Feeds page from being styled. Other creative solutions may exist, such as editing the My Feeds component to enable better selection, and so on.